tests/test_native_mcm.py

```diff
@@ -200,15 +199,15 @@ def obs_tape(x, y, z, reset=False, postselect=None):
     return m0, m1
 
 
-@flaky(max_runs=5)
+@pytest.mark.local_salt(42)
```

Comment:
What's the purpose of using local salt for these tests?

Reply:
Gotcha, so has this just been added to tests which give a "bad" result with the default salt?